src/main/java/org/docstr/gradle/plugins/gwt/GwtBasePlugin.java

```diff
@@ -89,15 +89,16 @@ public void apply(final Project project) {
         .setDescription("Classpath for GWT SDK libraries (gwt-dev, gwt-user)");
     allGwtConfigurations = project.files(gwtConfiguration, gwtSdkConfiguration);
 
-    addToMainSourceSetClasspath(allGwtConfigurations);
+    project.getConfigurations().getByName("implementation").extendsFrom(gwtConfiguration, gwtSdkConfiguration);
+    project.getConfigurations().getByName("testImplementation").extendsFrom(gwtConfiguration, gwtSdkConfiguration);

```

Comment:
Instead of hard-coding the names, can we please create constants at the beginning of the class, similar to other constants?